kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/RemoteKieFormsProvider.java

```diff
@@ -0,0 +1,140 @@
+package org.kie.server.services.jbpm.ui.form;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.enterprise.context.Dependent;
```

Comment:
could we avoid CDI annotations in kie server? and I guess Dependent is default if not defined, right?